### ISSUE TITLE
Swordmen instead than mounted units 

### ISSUE THREAD

**Issue by 0rion79** [RealBamboolord/LOTR-reworked#26]
hi, I don't know if is there anything interfering but I have noticed that everytime I try to build a mounted unit including the rohrrim, the game i strada gives me a swordman! I haven't noticed any other similar anomaly so far.

**Comment by RealBamboolord** [comment]
Yeah, that as reported a long time ago by some people, but I haven't be able to replicate it nor find the issue

**Comment by RealBamboolord** [comment]
Can you maybe try to update your Unciv?

**Comment by 0rion79** [comment]
hi, thanks. about my unciv it is already up to the latest version.

yes it is a weird bug, also because I think it didn't happened the very
first time I've played the mod! only from the 2nd time

Il lun 11 mar 2024, 17:28 Bamboolord ***@***.***> ha scritto:

**Comment by RealBamboolord** [comment]
Do you have a savefile for when this happens, so I can send it to the devs in a proper bug report?

**Comment by RealBamboolord** [comment]
Not necessary anymore, I looked at the code and found the problem (I'm curious how it slipped into the code). This should be resolved now if you update the mod

**Comment by 0rion79** [comment]
actually, the first game was flawless.... or was me who didn't notice?

then it showed up when I started a second time after understanding the
mechanics and the bug happens EVERY SINGLE TIME I try to build a rohrrim
knight: the game immediately turns it into a swordman.

I can properly build the war chariot (that I can upgrade to the ranged
rohrrim ) and the lancier, the knight that can be found later in the game
instead.

it's like there may be an error in the unit file that makes units obsolete
and indicates the possible upgrades.

btw, your mod reminds me a lot one that I built for Civ 3 many years ago.
There I had re-created the Eurasian continent, with crusades in Europe and
the raising of the mongol horde in the est.
in such occasion, I did fill the map with cities, preventing civs from
funding any more and created permanent alliances and hatred among tribes
while still allowing players to pick up their most beloved faction. I did
assign some of the Wonders to some cities to be conquered and others to be
built during the game. Since you've made such a cool map, it would be great
if you would turn your map into a mod such such characteristics!


Il lun 11 mar 2024, 20:07 Bamboolord ***@***.***> ha scritto:
